app/controllers/recipes_controller.rb

```diff
@@ -1,6 +1,13 @@
 class RecipesController < ApplicationController
+
   def index
-  end
+    if ingredient_id = params["ingredient"]
+      ingredient_id.to_i
```

Comment:
You're right, it works even if I delete this line at all. 
But it's really confusing - everything that comes from the internet is a string... But the page still works without .to_i anyway. Why?